nat-lab/tests/telio.py

```diff
@@ -1227,6 +1229,57 @@ async def probe_pmtu(self, host: str, expected: int):
         await self._write_command(["pmtu", host])
         await ev.wait()
 
+    # This is where natlab expects coredumps to be placed
+    # For CI and our internal linux VM, this path is set in our provisioning scripts
+    # If you're running locally without the aforementioned linux VM, you are expected to configure this yourself
+    # However, this only needs to be set iff you're: 
+    #  - running a test targeting a docker image
+    #  - have set the NATLAB_SAVE_LOGS environment variable
+    #  - want to have natlab automatically collect core dumps for you
+    def get_coredump_folder(self) -> tuple[str, str]: 
+        return "/var/crash", "core-"
+
+    def should_collect_core_dumps(self) -> bool:
```

Comment:
The function name seems to be reverse of what it actually returns, e.g.: `should_collect_core_dumps() == true` means that we should _skip_ collecting dumps

Reply:
fixed